core/code/ornaments.js

```diff
@@ -27,38 +39,105 @@ window.ornaments = {
       layerGroup = L.canvasIconLayer;
       L.CanvasIconLayer.mergeOptions({ padding: L.Canvas.prototype.options.padding });
     }
+    this.load();
     this._layer = layerGroup();
     this._beacons = layerGroup();
     this._frackers = layerGroup();
+    this._scout = layerGroup();
+    this._battle = layerGroup();
+    this._excluded = layerGroup(); // to keep excluded ornaments in an own layer
+
     window.layerChooser.addOverlay(this._layer, 'Ornaments');
     window.layerChooser.addOverlay(this._beacons, 'Beacons');
     window.layerChooser.addOverlay(this._frackers, 'Frackers');
+    window.layerChooser.addOverlay(this._scout, 'Scouting');
+    window.layerChooser.addOverlay(this._battle, 'Battle');
+    window.layerChooser.addOverlay(this._excluded, 'Excluded');
```

Comment:
```suggestion
    window.layerChooser.addOverlay(this._excluded, 'Excluded', {default: false});
```
Hiding excluded ornaments by default

Reply:
implemented